devU-api/src/entities/containerAutoGrader/containerAutoGrader.service.ts

```diff
@@ -107,12 +107,13 @@ export async function getGraderByAssignmentId(assignmentId: number){
     return {graderData, makefileData, autogradingImage, timeout}
 }
 
+
 export default {
     create,
     retrieve,
     update,
     _delete,
     list,
     getGraderByAssignmentId,
-    listByAssignmentId
+    listByAssignmentId,
```

Comment:
I left this service function here and added a new route to access it as an endpoint as I needed the ContainerAutoGrader object itself and needed to access it on the front end. I've renamed the listByAssignmentId function to make its purpose more clear. Would you suggest I modify the getGraderByAssignmentId function and make it accessible via the api instead of using a separate function?